src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -269,7 +269,6 @@ export const AzureComputeModalBase = ({
         ])
       }, [
         { label: 'Running cloud compute cost', cost: Utils.formatUSD(getAzureComputeCostEstimate(computeConfig)), unitLabel: 'per hr' },
-        // TODO [IA-3993] consider implementing pause via deallocate (which changes cost) instead of powerOff (which doesn't)
         { label: 'Paused cloud compute cost', cost: Utils.formatUSD(getAzureComputeCostEstimate(computeConfig)), unitLabel: 'per hr' },
```

Comment:
Hmm... the running cost should be different than the paused cost now, no?

Reply:
cc @mlilynolting here as well, I believe you looked into the cost differential a good bit? Is the cost-calculation updating captured in another ticket?

Reply:
The update to cost is not covered in another ticket, that is an AC of [IA-4105], this ticket. Paused cost should be $0 here.